File: jandy-python/jandy/ProfilingContext.py
from jandy.Serializer import Serializer


class ProfilingContext(Serializer):
    def __init__(self):
        self.profId = None
        self.threadObjects = None

    def getProfId(self):
        return self.profId

    def setProfId(self, profId):
        self.profId = profId
        return self

    def getThreadObjects(self):
        return self.threadObjects

    def setThreadObjects(self, threadObjects):
        self.threadObjects = threadObjects
        return self

    def toDict(self):
        return {
            'profId': self.profId,
            'threadObjects': self.threadObjects
        }

File: jandy-java-profiler/src/main/java/io/jandy/java/data/ProfilingContext.java
package io.jandy.java.data;

import java.util.List;

/**
 * @author JCooky
 * @since 2016-03-16
 */
public class ProfilingContext {
  private long profId;
  private List<ThreadObject> threadObjects;
  
  public long getProfId() {
    return profId;
  }

  public ProfilingContext setProfId(long profId) {
    this.profId = profId;
    return this;
  }

  public List<ThreadObject> getThreadObjects() {
    return threadObjects;
  }

  public ProfilingContext setThreadObjects(List<ThreadObject> threadObjects) {
    this.threadObjects = threadObjects;
    return this;
  }
}
